### PR TITLE
Add gtest test for fungible-token.

### PR DESCRIPTION
**Pull request by vivekvpandya**
This work is completely done by https://github.com/LouiseMedova Thanks!

### CODE REVIEW

File: fungible-token/src/lib.rs [hunk 1]
```diff
@@ -4,6 +4,9 @@
 #![no_std]
 #![feature(const_btree_new)]
 
+#[cfg(test)]
+mod tests;
```
Comment (shamilsan) [inline fungible-token/src/lib.rs:8]: Is it used anywhere?
```suggestion
```

Comment (shamilsan) [reply fungible-token/src/lib.rs:8]: > Is it used anywhere?

Sorry, confused it with integration tests.

File: fungible-token/src/tests.rs [hunk 2]
```diff
@@ -0,0 +1,322 @@
+use codec::Encode;
+use fungible_token_messages::*;
+use gstd::String;
+use gtest::{Program, System};
+const USERS: &'static [u64] = &[3, 4, 5];
+
+fn init_with_mint<'a>(sys: &'a System) {
```
Comment (shamilsan) [inline fungible-token/src/tests.rs:7]: Is lifetime really needed here?

```suggestion
fn init_with_mint(sys: &System) {
```

**Review by shamilsan — APPROVED** [review]
Looks great 👍

**Review by LouiseMedova — APPROVED** [review]

**Pull request merged**